.github/workflows/unit-testing-and-coverage.yml

```diff
@@ -89,7 +89,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ "macos-10.15", "ubuntu-18.04", "ubuntu-20.04", "windows-latest" ]
-        python-version: [ "3.6", "3.7", "3.8", "3.9", "pypy3.6" ]
+        python-version: [ "3.6", "3.7", "3.8", "3.9", "3.10", "pypy3.6" ]
```

Comment:
**question** should we drop python 3.6, since it's EOL? Also, is pypy3.6 EOL, or is that kinda the only thing on github CI? I guess we can still merge this, just wondering :shrug:

Reply:
We _can_, but really depends on version deprecation policy TBH. Since it is _so old_ at this point, I personally would be okay with dropping both 3.6 and pypy3.6 and adding support for maybe pypy3.7 or something more recent in a _minor release_, without waiting for 2.0 bump.

Reply:
I'm not sure if we have one. Arguably here we're only changing what we're /testing/ on... :shrug: I guess I don't feel strongly one way or the other :joy:

Reply:
> I'm not sure if we have one.

I guess people assume then that it defaults to Semver in that case :D If we _assume_ removing 3.6 support as deprecation, then it says we ought to emit deprecation warning in minor version and remove in next major version.

With limited resources and single branch dev, IME semver is impossible to follow. 

I'll create a separate ticket to discuss the test matrix, it could use updates on platforms we test on too, along with dropping 3.6 and possibly updating pypy3.7 or whatever the newest version it supports.

> Arguably here we're only changing what we're /testing/ on... 🤷

True, but we can only officially advertise support on versions we actually are testing on. _Technically_ You can run aiosmtpd on Python 3.10 (which some Mailman users are doing, because there isn't an upper bound for Python version in last published version of aiosmtpd or Mailman so `pip` won't complain) and it would be okay in case of Mailman since we use only LMTP with no SSL (which is where most of the issues with 3.10 compat are ;-)

Reply:
fair, guess we can go ahead and merge this and just continue our discussion in the new issue 👍